task/jefe/src/main.rs

```diff
@@ -252,6 +253,20 @@ impl idol_runtime::NotificationHandler for ServerImpl<'_> {
                 }
             }
         }
+        // Check whether all 'exit' tasks have 'Exited'
+        let mut done: bool = true;
+        for (i, exits) in TASK_EXITS.iter().enumerate() {
+            if *exits
+                && (kipc::read_task_status(i)
+                    != abi::TaskState::Healthy(abi::SchedState::Exited))
```

Comment:
Style: Maybe this can be included as a branch at §243, and just match against TASK_EXITS - rather than another iterator loop across task statuses.